app/components/candidate_interface/other_qualifications_review_component.html.erb

```diff
@@ -38,8 +38,14 @@
   <% else %>
     <% if @editable %>
       <%= govuk_inset_text(classes: 'govuk-!-width-two-thirds govuk-!-margin-top-0') do %>
-        <p class="govuk-body">Adding A levels and other qualifications makes your application stronger. They demonstrate subject knowledge not covered in your degree or work experience.</p>
-        <p class="govuk-body">Training providers usually ask you for them later in the process.</p>
+        <% if application_form.teacher_degree_apprenticeship_feature_active? %>
+          <p class="govuk-body"><%= t('.a_levels_are_required') %></p>
+          <p class="govuk-body"><%= t('.a_levels_postgraduate_or_undergraduate') %></p>
+        <% else %>
+          <p class="govuk-body"><%= t('.a_levels_postgraduate') %></p>
+        <% end %>
+
+        <p class="govuk-body"><%= t('.training_providers') %></p>
```

Comment:
There's a problem using the `.key` syntax in component previews. I'm not sure why, it requires investigating.

I think we should aim to have previews for our components, and this will block a preview for this. 
Can we use the full translation scope here for now?

or set a local variable:

```ruby
scope = 'other_qualifications_review_component'
t('a_levels_are_required', scope:)
```

Reply:
I added the preview for this component and it worked (for postgraduate and undergraduate):

![screencapture-localhost-3000-rails-view-components-candidate-interface-other-qualifications-review-component-no-a-levels-2024-09-05-06_39_45](https://github.com/user-attachments/assets/5edfc9bd-9833-4630-ab0b-ab31e228f9f3)

Reply:
Added the preview on 66a6cfd0a297777196d22f5ca3a34a8b228f4532

Reply:
That's weird. I thought it _should_ work but thought it didn't.